draft-ietf-quic-transport.md

```diff
@@ -200,7 +200,8 @@ Out-of-order packet:
 
 : A packet that does not increase the largest received packet number for its
   packet number space ({{packet-numbers}}) by exactly one. A packet can arrive
-  out of order if it is delayed or if earlier packets are lost or delayed.
+  out of order if it is delayed, if earlier packets are lost or delayed, or if
+  the sender intentionally skips a packet number.
```

Comment:
Reading through the whole definition now, I'm wondering why an out-of-order packet is one that doesn't increase the largest received packet number. If a packet jumps ahead of the queue, I would have considered it out-of-order, but it's not according to this definition.

Reply:
@marten-seemann If the received packet number is not an increment by one over the largest received, it is considered out-of-order. This also applies to packets that jumps ahead of the queue.

Reply:
Sorry, missed the "by exactly one".